Mathlib/SetTheory/Cardinal/PartENat.lean

```diff
@@ -82,15 +82,15 @@ lemma toPartENat_le_iff_of_le_aleph0 {c c' : Cardinal} (h : c ≤ ℵ₀) :
   lift c to ℕ∞ using h
   simp only [← partENatOfENat_toENat, toENat_ofENat, enat_gc _,
    ← PartENat.withTopOrderIso.symm.le_iff_le]
-  rfl
+  simp only [PartENat.ofENat_le, map_le_map_iff]
```

Comment:
Is there a reason to have two `simp only` in a row? Can this just be changed to `simp` since it's the end of a proof?